hack/generate-gpu-count-table.sh

```diff
@@ -34,4 +35,4 @@ function get_all_gpus() {
   done
 }
 
-get_all_gpus | sort | uniq
+get_all_gpus | sort | uniq | grep -vE "(g5.48xlarge)"
```

Comment:
```suggestion
get_all_gpus | sort | uniq | grep -v "g5.48xlarge"
```

Is this slightly simpler version possible instead? (ditto with other file)

Reply:
```
❯ vim test.txt
❯ cat test.txt
"g5.48xlarge": 8,
"g5.43xlarge": 8,
❯ grep -v "(g5.48xlarge)" test.txt
"g5.48xlarge": 8,
"g5.43xlarge": 8,
❯ grep -vE "(g5.48xlarge)" test.txt
"g5.43xlarge": 8,
```

unfortunately no